docs/DeveloperGuide.md

```diff
@@ -300,11 +549,56 @@ Use case ends.
 
 > Windows, Linux, Unix, OS-X
 
-##### Private contact detail
+## Appendix E : Product Survey
```

Comment:
Wow @team, in general I do like how u guys tried to survey the products with the approach of how you can compare it to the features/functionalities in the app you are building! Great Job! This is how product surveys should be done!!!

However since this is to be added into the report I do urge you guys to add a few more observations ~